root-common/usr/bin/run-mysqld-slave

```diff
@@ -14,6 +14,11 @@ export_setting_variables
 
 log_volume_info $MYSQL_DATADIR
 
+# Just run normal server if the data directory is already initialized
+if [ -d "${MYSQL_DATADIR}/mysql" ]; then
```

Comment:
@hhorak - turns out this change is breaking the mysql replica tests in https://github.com/openshift/origin/blob/master/test/extended/image_ecosystem/mysql_replica.go ... we've had to comment out that  test since this change dropped.

In our test, we force a restart of the slave pod by deleting the existing slave pod.  On the pod restart, this directory is discovered and we run `run-mysqld`, without setting `MYSQL_RUNNING_AS_SLAVE`.  As such, the process then attempts to validate various env vars in the `20-validate-variables.sh` script, and the slave restart fails.

How would you propose that slave restarts work when a PV is present with this change?

@bparees